testeng/resources/cleanup-python-code-create-pr.sh

```diff
@@ -1,34 +1,49 @@
 #!/bin/bash
 set -e
 
-rm -rf code_cleanup_venv
-virtualenv --python=python$PYTHON_VERSION code_cleanup_venv -q
-source code_cleanup_venv/bin/activate
+declare -i RESULT=$?
 
-echo "Upgrading pip..."
-pip install pip==20.0.2
+IFS=',' read -ra SCRIPTSTORUN <<<"$SCRIPTS"
 
-echo "Getting current sha..."
-cd $REPO_NAME
-export CURRENT_SHA=$(git rev-parse HEAD)
+PACKAGESTOINSTALL=$(echo $PACKAGES | tr , " ")
 
-echo "Running cleanup..."
+IFS=',' read -ra REPOS <<<"$REPO_NAMES"
 
-PACKAGESTOINSTALL=`echo $PACKAGES | tr , " "`
+for repo in "${REPOS[@]}"; do
 
-pip install $PACKAGESTOINSTALL
+  rm -rf "${repo}"
 
-IFS=',' read -ra SCRIPTSTORUN <<< "$SCRIPTS"
-for element in "${SCRIPTSTORUN[@]}"; do
-  `echo "$element"`
-done
+  if git clone https://github.com/edx/"${repo}".git; then true; else RESULT=$? && continue; fi
```

Comment:
Could we also build a list of which repos failed, and give it at the end?  For a nontrivial number of repos, it can be pretty hard to inspect the log and figured out how many and which repos failed.